client/new-stack-editor/lib/index.coffee

```diff
@@ -47,3 +62,15 @@ module.exports = class StackEditorAppController extends AppController
     console.trace()
     log '::reloadEditor', templateId, skipDataUpdate
 
+
+  fetchStackTemplate: (templateId, callback) ->
+
+    if template = @templates[templateId]
+      return callback null, template
+
+    cc = kd.singletons.computeController
+    cc.fetchStackTemplate templateId, (err, template) =>
```

Comment:
/cc @hakankaradis 

We need to be aware that this instance will not go into the flux stores.